src/sidebar/services/test/session-test.js

```diff
@@ -39,10 +36,10 @@ describe('sidebar/services/session', function () {
         currentProfile = newProfile;
       }),
     };
-    fakeAuth = {
+    fakeAuth = Object.assign(new EventEmitter(), {
```

Comment:
Having to extend `EventEmitter` in this tests feels just a touch fiddly...

Reply:
We could mock the individual methods (`on`, `emit`) but IME it tends to be easier just to use a real `EventEmitter`.